kyo-scheduler/jvm/src/main/scala/kyo/scheduler/Worker.scala

```diff
@@ -90,18 +91,19 @@ final private class Worker(
         available
     }
 
-    def isStalled(cycles: Long): Boolean =
+    private def isStalled(cycles: Long): Boolean =
         running && currentCycle < cycles - 2
 
-    def isBlocked(): Boolean = {
-        val mount = this.mount
-        mount != null && {
-            val state = mount.getState().ordinal()
-            state == Thread.State.BLOCKED.ordinal() ||
-            state == Thread.State.WAITING.ordinal() ||
-            state == Thread.State.TIMED_WAITING.ordinal()
+    private def isBlocked(): Boolean =
+        running && {
```

Comment:
a minor unrelated fix: the top utility was incorrectly reporting inactive workers as blocked.